packages/app-project/stores/User/UserPersonalization/Notifications/Notifications.js

```diff
@@ -47,11 +67,18 @@ const Notifications = types
       }),
 
       processSugarNotification (notification) {
-        // sugar data objects with source_type = 'Message' are related to Talk messages/conversations and included in ZooHeader Messages, not in ZooHeader Notifications
+        // sugar data objects with source_type = 'Message' are related to Talk messages, a subset of Talk conversations, and are included in ZooHeader Notifications
         // sugar data objects with source_type = 'Moderation' are related to Talk moderation reports and are not included in ZooHeader Notifications or ZooHeader Messages
 
-        if (notification?.data.source_type !== 'Message' && notification?.data.source_type !== 'Moderation') {
-          self.count = self.count + 1
+        if (notification?.data.source_type !== 'Moderation') {
+          self.unreadNotificationsCount = self.unreadNotificationsCount + 1
+        }
```

Comment:
This will increment if `notification?.data.source_type === 'Message'`. Is that right? In the deleted line, messages are ignored.

Reply:
That's correct, I realized if you get a new message for a new conversation it increments your Notification count by one and your Messages count by one. If you leave the conversation (i.e. Message in header) unread and get an additional message in that same conversation, it'll increment your Notification by one but leave your Message count as is (the same single conversation is still unread, but now you have 2 unread messages/Notifications related to that conversation).